vm/qemu/qemu.go

```diff
@@ -146,7 +146,7 @@ var archConfigs = map[string]*archConfig{
 	},
 	"linux/arm64": {
 		Qemu:     "qemu-system-aarch64",
-		QemuArgs: "-machine virt,virtualization=on -cpu cortex-a57 -accel tcg,thread=multi",
+		QemuArgs: "-machine virt,virtualization=on -machine gic-version=max -cpu cortex-a57 -accel tcg,thread=multi",
```

Comment:
This change is almost a no-op, as we do not set up any interrupt controllers so far.

Reply:
Are the "machine" flags added, or later override the previous ones?
May be safer to do: `-machine virt,virtualization=on,gic-version=max`.

Reply:
No idea :)
Ack